src/inference/pose_result.py

```diff
@@ -65,11 +65,11 @@ def to_numpy_ma(self) -> np.ndarray:
             
         # Get dimensions
         num_frames = len(self.frames)
-        max_persons = max(len(frame.persons) for frame in self.frames)
+        max_persons = max(len(frame.persons) if frame.persons else 0 for frame in self.frames)     
         num_keypoints = max(
             len(person.keypoints)
             for frame in self.frames
-            for person in frame.persons
+            for person in frame.persons if frame.persons
```

Comment:
These changes should not be necessary anymore, as with the latest PR #106 I made sure, that frame.persons is never None.